chia/util/keyring_wrapper.py

```diff
@@ -100,57 +67,24 @@ class KeyringWrapper:
 
     # Instance members
     keys_root_path: Path
-    keyring: Union[Any, FileKeyring] = None
+    keyring: FileKeyring
     cached_passphrase: Optional[str] = None
     cached_passphrase_is_validated: bool = False
-    legacy_keyring = None
 
-    def __init__(self, keys_root_path: Path = DEFAULT_KEYS_ROOT_PATH, force_legacy: bool = False):
+    def __init__(self, keys_root_path: Path = DEFAULT_KEYS_ROOT_PATH):
         """
-        Initializes the keyring backend based on the OS. For Linux, we previously
-        used CryptFileKeyring. We now use our own FileKeyring backend and migrate
-        the data from the legacy CryptFileKeyring (on write).
+        Initializes the keyring backend.
         """
-        from chia.util.errors import KeychainNotSet
 
         self.keys_root_path = keys_root_path
-        if force_legacy:
-            legacy_keyring = get_legacy_keyring_instance()
-            if legacy_keyring is not None and check_legacy_keyring_keys_present(legacy_keyring):
-                self.legacy_keyring = legacy_keyring
-        else:
-            self.refresh_keyrings()
-
-        if self.keyring is None and self.legacy_keyring is None:
-            raise KeychainNotSet(
-                f"Unable to initialize keyring backend: keys_root_path={keys_root_path}, force_legacy={force_legacy}"
-            )
-
-    def refresh_keyrings(self):
-        self.keyring = None
         self.keyring = self._configure_backend()
 
-        # Configure the legacy keyring if keyring passphrases are supported to support migration (if necessary)
-        self.legacy_keyring = self._configure_legacy_backend()
-
         # Initialize the cached_passphrase
         self.cached_passphrase = self._get_initial_cached_passphrase()
 
     def _configure_backend(self) -> FileKeyring:
-        if self.keyring:
-            raise Exception("KeyringWrapper has already been instantiated")
         return FileKeyring.create(keys_root_path=self.keys_root_path)
```

Comment:
Are we keeping this method because the test code monkeypatches it?

Reply:
(that wasn't quite the code I wanted to highlight). I wouldn't block on this, just curious.

Reply:
Yeah right, its because of the monkeypatch. I tried to get rid of it but its a bit more involved then and i wanted to leave any actual changes out of here.